src/View/User/userRanking.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}Classement
+{% endblock %}
+
+{% block stylesheet %}
+	{{parent()}}
+	<link rel="stylesheet" href="/assets/css/ranking.css">
+{% endblock %}
+
+{% block content %}
+	<main>
+		<div class="podium">
+			{% for user in users|slice(0,3) %}
+				<div class="rank">
+					<figcaption>{{user.name}}</figcaption>
+					<img src="/assets/images/{{user.avatar}}" alt="{{user.name}}"/>
+					<div class="podium-{{loop.index}}">
+						<img src="/assets/images/podium1.png"/>
+					</div>
+				</div>
+			{% endfor %}
+		</div>
+		<table>
+			<tr>
+				<th></th>
+				<th>Rang</th>
+				<th>Pseudo</th>
+				<th>Points</th>
+			</tr>
+			{% set rank = 1 %}
+			{% for user in users %}
+				<tr>
+					<td><img src="/assets/images/{{user.avatar}}"/></td>
+					<td>{{rank}}</td>
+					<td class="left-justify">{{user.name}}</td>
```

Comment:
in the updated database it was nickName right? just need to change this so it display the nickname